app/src/editor/canvas/components/ModuleDragger.jsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 
 import { updateModulePosition } from '../state'
 import { Draggable } from './DragDropContext'
 import ModuleStyles from '$editor/shared/components/Module.pcss'
+import * as CanvasController from './CanvasController'
```

Comment:
Should we just `import { useController } from './CanvasController'` as it is used only in one place?

Reply:
Sure. Done in 4379848.